api/src/services/dashboard/todo.service.js

```diff
@@ -50,6 +50,10 @@ service.todosByRole = async (user) => {
     case ROLES.SUPERVISOR:
     case ROLES.RESPONSIBLE:
     case ROLES.HEAD_CENTER:
+      functionsByRole = {
+        inscription: [DASHBOARD_TODOS_FUNCTIONS.INSCRIPTION.IMAGE_RIGHT],
+        sejour: [DASHBOARD_TODOS_FUNCTIONS.SEJOUR.DOCS, DASHBOARD_TODOS_FUNCTIONS.SEJOUR.CHECKIN, DASHBOARD_TODOS_FUNCTIONS.SEJOUR.YOUNG_TO_CONTACT],
```

Comment:
To merge en une colonne

Reply:
je peux pas sinon dans la fonction : 
 const arr = await Promise.all(
      value.map((f) =>
        services[key][f](user, {.......
      services[key][f] est undefined car la fonction IMAGE_RIGHT n'existe pas dans la clé sejour. 
      Je peux réarranger dans le front ou c'est chiant ?